melodies_monet/driver.py

```diff
@@ -492,17 +492,20 @@ def glob_files(self):
         from . import tutorial
 
         print(self.file_str)
-        if self.file_str.startswith("example:"):
+        if isinstance(self.file_str, list):
+            self.files = self.file_str # we assume the given list is already sorted
```

Comment:
```suggestion
            self.files = sorted(self.file_str)
```
Is there a reason not to do this?
I am not too happy with the name file_str if it can be a list, but I guess that would require further changes to the code. Would that be doable?

Reply:
Not at all. I think adding the sorted() would be better than inputting an already sorted list. I'll add commit this suggestion. 

As for the file_str, I could rename this in the driver. What would you suggest renaming it to?